<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_book</class>
 <widget class="QDialog" name="add_book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>74</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.PingFang SC0</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>标题</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>160</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.PingFang SC0</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>评级</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>203</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.PingFang SC0</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版社</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>117</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.PingFang SC0</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作者</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>246</y>
     <width>35</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.PingFang SC0</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ISBN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>289</y>
     <width>34</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.PingFang SC0</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>页数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.PingFang SC0</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>图书</string>
   </property>
  </widget>
  <widget class="QLabel" name="as_title">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>74</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 89, 92);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="as_author">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>117</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 89, 92);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="indent">
    <number>-5</number>
   </property>
  </widget>
  <widget class="QLabel" name="as_pre">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>203</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 89, 92);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="as_isbn">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>246</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 89, 92);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="as_page">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>289</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 89, 92);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_press">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>205</y>
     <width>152</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_isbn">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>248</y>
     <width>152</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_pages">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>291</y>
     <width>152</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_title">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>76</y>
     <width>152</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_author">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>119</y>
     <width>152</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="box_lev">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>162</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>未评级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>一般</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>成人</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>儿童</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="as_check">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_can">
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>371</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>line_title</tabstop>
  <tabstop>line_author</tabstop>
  <tabstop>box_lev</tabstop>
  <tabstop>line_press</tabstop>
  <tabstop>line_isbn</tabstop>
  <tabstop>line_pages</tabstop>
  <tabstop>btn_can</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
